composables/useAsset.ts

```diff
@@ -0,0 +1,11 @@
+import { useAssetsStore } from '~/stores/assets'
+
+export default function () {
+  const assetIdOf = (id: string | number) => {
+    return useAssetsStore().getAssetById(String(id))
+  }
+
+  return {
+    assetIdOf,
+  }
+}
```

Comment:
We have already something like this `const { assets } = usePrefix()`

Reply:
Thanks for pointing that out @roiLeo, i just migrated the assetsMixin to composable. And since there are no useAsset i assumed no one migrated the mixin before. Good thing it is in usePrefix now so we can remove this composable.

Thanks

Reply:
yes please remove, it throw a deepscan issue

Reply:
Thanks, will do. Btw where do you find the deepscan issue? Would love to have that on my other repos

Reply:
> Thanks, will do. Btw where do you find the deepscan issue? Would love to have that on my other repos

eheh sorry I was refering about https://github.com/kodadot/nft-gallery/pull/6693#discussion_r1292886024

Deepscan can be found in actions 
![Screenshot 2023-08-14 at 08-47-25 Refactor_decorator2 by stephenjason89 · Pull Request #6693 · kodadot_nft-gallery](https://github.com/kodadot/nft-gallery/assets/9987732/0adaa689-a206-43c0-b43a-3863bb311f52)